init.rb

```diff
@@ -1,14 +1,12 @@
-# Global Hooks
-require File.expand_path('../lib/redmine_gtt/hooks/view_layouts_base_html_head_hook', __FILE__)
-require File.expand_path('../lib/redmine_gtt/view_hooks', __FILE__)
+require_relative 'lib/redmine_gtt/view_hooks'
 
 Redmine::Plugin.register :redmine_gtt do
   name 'Redmine GTT plugin'
   author 'Georepublic'
   author_url 'https://github.com/georepublic'
   url 'https://github.com/gtt-project/redmine_gtt'
   description 'Adds location-based task management and maps'
-  version '5.1.2'
```

Comment:
@sanak , I thought we need a major version change when supporting Redmine 6. But as you could make it work with Redmine 5, too, maybe a major version is not needed ... or even a bad idea? What do you think?

What is necessary though is selecting suitable gem versions.
So what would be an appropriate version change?

Reply:
@dkastl Well, I think that major version change is still good, because Redmine core side affects a lot to this plugin. Also, it will be nice to have same major version with Redmine. 😊

Reply:
OK, I agree.
Shall we maybe stop supporting v5.0 and make 5.1 the minimum version requirement? This would be a breaking change then. And I tink you also removed it frome the tests.

Reply:
Yes, dropping support of v5.0 is reasonable for me.